roles/KubevirtNodeLabeller/defaults/main.yml

```diff
@@ -2,12 +2,3 @@
 # defaults file for KubevirtNodeLabeller
 wait: true
 use_kvm: true
-
-kubevirt_node_labeller_image: "node-labeller"
```

Comment:
I like it when the role is working when executed manually. I commonly treat the defaults file as docs for the role too. So personally I would keep generic defaults here.

Reply:
OK, so move back the image names per-role and have the versions in the top-level `_defaults.yml`, plus adding few lines of doc explaining how it will work (= summarizing the discussion we're having here). Would that work?

Reply:
Yeah, keep the image names and versions here, but change all versions here to latest and explain that the vars here are just for manual execution purposes. Release related defaults should be set in the _defaults.yml file.

Reply:
ack, done